dataAccessObjects.py

```diff
@@ -196,6 +196,16 @@ def generateNewId(self):
         else:
             return "pref0000000001"
 
+    def getNowUserPrefer(self,dataType,UserId):
+        sql = 'SELECT pref_data_type_{dataType} FROM user_prefer WHERE pref_is_delete=0 and user_id={UserId}' \
```

Comment:
Have you tried to run the arrange_schedule.py after modified? I think this change will cause error in these two lines.